.github/workflows/reusable-run-linting-check-and-unit-tests.yml

```diff
@@ -7,6 +7,31 @@ permissions:
   contents: read
 
 jobs:
+  code-quality:
```

Comment:
It's not mandatory to accept my suggestion, but does it make sense to add a comment during the transition? If another maintainer or community wants to help with this, it will make it easier to identify what this new work does.

Reply:
Good point, I have added documentation for the workflow and explained the current migration. Thanks!